docs/common-problems.asciidoc

```diff
@@ -177,3 +178,37 @@ This is not a SSL problem. Make sure that Logstash is running and that there is
 
 A firewall is refusing the connection. Check if a firewall is blocking the traffic on the client, the network, or the
 destination host.
+
+[[field-limit-exceeded]]
+[float]
+=== Field limit exceeded
+
+When adding too many distinct tag keys on a transaction or span, 
+you risk creating a link:{ref}/mapping.html#mapping-limit-settings[mapping explosion].
+
+For example,
```

Comment:
i know is not introduced here, but i'd appreciate if we stop doing line breaks systematically after punctuation marks. it makes docs harder to read for no good reason.

Reply:
I'd like to defer this discussion to general docs discussions and not just break with our internal guidelines here.